src/zenml/integrations/mlflow/model_deployers/mlflow_model_deployer.py

```diff
@@ -0,0 +1,311 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+import json
+import os
+import shutil
+from typing import Optional
+
+from typing import List, Optional
+
+from zenml import logger
+from zenml.constants import LOCAL_STORES_DIRECTORY_NAME
+
+from zenml.enums import ModelDeployerFlavor, StackComponentType, ModelDeployerFlavor
+from zenml.integrations.mlflow.services.mlflow_deployment import (
+    MLFlowDeploymentConfig,
+    MLFlowDeploymentService,
+)
+from zenml.io.utils import get_global_config_directory
+from zenml.model_deployers.base_model_deployer import BaseModelDeployer
+from zenml.services import (
+    BaseService, 
+    ServiceRegistry
+)
+from zenml.services.local.local_service import SERVICE_DAEMON_CONFIG_FILE_NAME
+from zenml.stack.stack_component_class_registry import register_stack_component_class
+
+
+@register_stack_component_class(
+    component_type=StackComponentType.MODEL_DEPLOYER,
+    component_flavor=ModelDeployerFlavor.MLFLOW,
+)
+class MLFlowModelDeployer(BaseModelDeployer):
+    """MLflow implementation of the BaseModelDeployer"""
+
+    @property
+    def type(self) -> StackComponentType:
+        """The component type."""
+        return StackComponentType.MODEL_DEPLOYER
+
+    @property
+    def flavor(self) -> ModelDeployerFlavor:
+        """The model deployer flavor."""
+        return ModelDeployerFlavor.MLFLOW
+
+    def deploy_model(
+        self,
+        config: MLFlowDeploymentConfig,
```

Comment:
What if we allow this to be used even outside the "continuous model deployment" setting ? In other words, what if we had a `replace: bool = *` parameter that turns off the "stop the older model deployment service first" behavior ?